execute/transport.go

```diff
@@ -654,8 +654,8 @@ func (t *transformationTransportAdapter) ProcessMessage(m Message) error {
 
 		// Retrieve the buffered builder for the given key
 		// and send the data to the next transformation.
-		tbl, err := t.cache.Table(m.Key())
-		if err != nil {
+		tbl, ok, err := t.cache.Table(m.Key())
+		if err != nil || !ok {
```

Comment:
Based on the implementation of `ProcessChunk` wouldn't we want to produce an error? Getting a flush key message for something not in the table cache would seem to indicate an error somewhere. Do you have an example where this might be expected?

Reply:
I found this when I was working on implementing filter. Filter is a narrow transformation so it passes the flush key message. If filter completely filters the table, it won't produce any chunks for a certain key. But, it still passes the flush key message.

It does seem a bit strange to me that this is possible, but in order for filter to not forward this message, it would have to keep state of every group key it had seen and sent to know whether it should forward the flush key or swallow it. I think the cost of making filter do accounting of all of the group keys it has seen would probably be worse than just dropping a message that should probably not have been passed.